src/script/interpreter.cpp

```diff
@@ -305,8 +306,6 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
     {
         while (pc < pend)
         {
-            bool fExec = !count(vfExec.begin(), vfExec.end(), false);
```

Comment:
value of fExec may change only if an IF, NOTIF, ELSE, or ENDIF is encountered. No other opcode may flip `fExec`. So we don't need to redo it for every opcode, just after IF, NOTIF, ELSE, or ENDIF